ios/ReactTestApp/AppDelegate.swift

```diff
@@ -23,6 +23,12 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
                      didFinishLaunchingWithOptions _: [UIApplication.LaunchOptionsKey: Any]?) -> Bool
     {
         self.application = application
+
+        NotificationCenter.default.post(
+            name: .ReactTestAppDidInitialize,
+            object: nil
+        )
+
         return true
```

Comment:
Can we put this in a `defer` block so it gets called after returning?
```suggestion
        defer {
            NotificationCenter.default.post(
                name: .ReactTestAppDidInitialize,
                object: nil
            )
        }

        self.application = application
        return true
```

Reply:
Done. Out curiosity, why?

Reply:
Technically, the app isn't really initialized until after you've returned `true` here.